QuizNet.BusinessLogic/QuizService.cs

```diff
@@ -49,5 +49,15 @@ public int CheckQuiz(List<QuestionDto> questions, int[] userAnswers)
 
             return correctAnswers;
         }
+
+        public List<QuestionDto> GenerateRecentlyAddedQuestionsQuiz()
+        {
+            var questions = _questionRepository.GetAll().ToList();
+            var randomQuestions = questions.OrderByDescending(x => x.CreationTime).Take(3).ToList();
```

Comment:
Czy aby na pewno ta zmienna powinna nazywać się 
`var randomQuestions` ?

Reply:
Pewnie, że nie, zmienię to.